src/datasources/locking-api/locking-api.service.ts

```diff
@@ -24,13 +24,34 @@ export class LockingApi implements ILockingApi {
       this.configurationService.getOrThrow<string>('locking.baseUri');
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async getRank(safeAddress: `0x${string}`): Promise<Rank> {
+    try {
+      const url = `${this.baseUri}/api/v1/leaderboard/${safeAddress}`;
+      console.log('=>', url);
+      const { data } = await this.networkService.get<Rank>(url);
+      return data;
+    } catch (error) {
+      console.log('=>', error);
```

Comment:
nit: `console.log` left here.

Reply:
Removed in afd4cf9cfcfa0b968c0de7d34b12b56b524c6410.